config/webpack.config.prod.js

```diff
@@ -123,26 +131,48 @@ module.exports = {
         // Webpack 1.x uses Uglify plugin as a signal to minify *all* the assets
         // including CSS. This is confusing and will be removed in Webpack 2:
         // https://github.com/webpack/webpack/issues/283
-        loader: ExtractTextPlugin.extract('style', 'css?-autoprefixer!postcss'),
         // Note: this won't work without `new ExtractTextPlugin()` in `plugins`.
+
+        // Note: ExtractTextPlugin is deprecated for webpack > 2, MiniCssExtractPlugin used instead
+        use: [
+          {
+            loader: MiniCssExtractPlugin.loader
+          },
+          'css-loader?-autoprefixer!postcss-loader'
+        ]
       },
       {
         test: /\.css$/,
         exclude: /(node_modules)|(-nomodules\.css$)/,
-        loader: 'style!css?modules!postcss',
+        use: [
+          {
+            loader: 'style-loader'
+          },
+          {
+            loader: 'css-loader',
+            options: {
+              modules: true
+            }
+          },
+          {
+            loader: 'postcss-loader'
+          }
+        ]
       },
+
       // JSON is not enabled by default in Webpack but both Node and Browserify
       // allow it implicitly so we also enable it.
       {
         test: /\.json$/,
-        loader: 'json',
+        loader: 'json-loader',
```

Comment:
I don't think this loader is needed anymore

Reply:
Right. I can see I removed that in _config/webpack.config.dev.js_ and forgot to do it also here. Thank you! 👍🏽